server/i18n/en.json

```
@@ -8090,6 +8102,22 @@
     "id": "ent.migration.migratetosaml.username_already_used_by_other_user",
     "translation": "Username already used by another Mattermost user."
   },
+  {
+    "id": "ent.outgoing_oauth_connections.delete_connection.app_error",
+    "translation": "There was an error while deleting the connection."
+  },
+  {
+    "id": "ent.outgoing_oauth_connections.get_connections.app_error",
+    "translation": "There was an error while getting the connections."
```

This could read vague if this admin thinks this means "their computer's connection to the Mattermost server"

```suggestion
    "translation": "There was an error while getting the outgoing oauth connections."
```